Jenkinsfile

```diff
@@ -71,8 +71,8 @@ pipeline {
       steps {
         pipelineManager([ cancelPreviousRunningBuilds: [ when: 'PR' ] ])
         deleteDir()
-        gitCheckout(basedir: "${BASE_DIR}", githubNotifyFirstTimeContributor: true, shallow: false,
-                    reference: '/var/lib/jenkins/.git-references/apm-agent-java.git')
+        // reference repo causes issues while running on Windows with the git-commit-id-maven-plugin
```

Comment:
[question] the `reference` option seems to be an optimization for faster clone/checkout, is there any noticeable impact here ? If there is one, then maybe using a separate checkout on Windows with the same commit ID could be an option.